packages/oraidex-sync/src/db.ts

```diff
@@ -583,18 +587,6 @@ export class DuckDb {
     return result as Pick<PoolApr, "apr" | "pairAddr" | "rewardPerSec" | "totalSupply">[];
```

Comment:
should move to declaration of function():Promise<Pick<PoolApr, "apr" | "pairAddr" | "rewardPerSec" | "totalSupply">[]>